db/compaction_history_entry.hh

```diff
@@ -3,26 +3,37 @@
  */
 
 /*
- * SPDX-License-Identifier: AGPL-3.0-or-later
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
```

Comment:
This should have been in the patch that introduced the new file, not here.

Reply:
Fixed. Moved to the correct patch.